src/pages/Stopwatch/page.tsx

```diff
@@ -0,0 +1,5 @@
+import { Stopwatch } from './components/Stopwatch';
+
+const StopWatchPage = () => <Stopwatch />;
+
+export default StopWatchPage;
```

Comment:
3번째 라인을 지우고 다음과 같이 하셔도 됩니다. 이를 re-export 라고 합니다.

```suggestion
export default Stopwatch
```